.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+
+os:
+  - osx
+  - linux
+
+language: node_js
+node_js: lts/*
+
+before_install:
+  - if [ $TRAVIS_OS_NAME == "linux" ]; then
```

Comment:
We probably don't need any of this.

We don't use any native modules, so `CXX` and `CC` are not needed.

I doubt need X either.

Reply:
These were for running tests, which we aren't doing.